test cases/common/38 string operations/meson.build

```diff
@@ -101,3 +101,11 @@ assert('\\\\n' == bs_bs_n, 'Four backslash broken before n')
 assert('\\\\\n' == bs_bs_nl, 'Five backslash broken before n')
 assert('\\\\' == bs_bs, 'Double-backslash broken')
 assert('\\' == bs, 'Backslash broken')
+
+mysubstring='foobarbaz'
+assert(mysubstring.substring() == 'foobarbaz', 'substring is broken')
+assert(mysubstring.substring(0) == 'foobarbaz', 'substring is broken')
+assert(mysubstring.substring(1) == 'oobarbaz', 'substring is broken')
+assert(mysubstring.substring(-5) == 'foobarbaz', 'substring is broken')
```

Comment:
What's the rationale for allowing negative `start` values? Why not just error out?

Reply:
because i wanted to avoid errors and I followed javascript sbstring doc (not sure its a good argument :p)

@xclaesse I can implement, so negative end value means pos = len - value right ?